src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -72,27 +76,36 @@ abstract class GeneratorCommand extends Command
         'include_once',
         'instanceof',
         'insteadof',
+        'int',
         'interface',
         'isset',
+        'iterable',
         'list',
+        'match',
```

Comment:
mixed is also reserved

Reply:
Thanks for the review!

As you said, `mixed` seems to work as a reserved word as well.
I also found a PR in the language documentation.
https://github.com/php/doc-en/pull/938

Likewise, it looks like `never` will be added in PHP 8.1!
 - Class https://3v4l.org/8sXc6
 - Interfaces https://3v4l.org/M0RSh
 - trait https://3v4l.org/tEIT2